hugolib/pageSort.go

```diff
@@ -275,3 +276,20 @@ func (p Pages) Reverse() Pages {
 
 	return pages
 }
+
+func (p Pages) ByParam(paramsKeyStr string) Pages {
```

Comment:
Make `paramsKeyStr` an `interface{}`-- it comes from the templates, and it *may* be something else.